src/lint/rules/tests/spacing/mod.rs

```diff
@@ -55,9 +54,10 @@ typedef layout \"little-endian\" {bitfields 8 {uint2 rsvd @ [7:6];
 fn style_check_sp_braces_02() {
     let mut cfg = LintCfg::default();
     let mut rules = instantiate_rules(&cfg);
-    assert_snippet(SP_BRACES_02, 6, &rules);
+    assert_snippet(SP_BRACES_02, 8, &rules);
     // Test rule disable
     cfg.sp_brace = None;
+    cfg.in4 = None;
     rules = instantiate_rules(&cfg);
```

Comment:
Update SP_BRACES_02 to be a single line with a single bit field, so that closing braces don't trigger in4 here.
Turn the expected amount of errors back to 6.